### PR TITLE
formatted html, removed spacs, removed comments from css file

### PR DESCRIPTION
**Pull request by maosan132**
Finished some minor changes, cleaning up code, removed empty lines and comment lines in the code of the CSS and Html files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
+                <li class="qbooks">quickbooks</li>
+                <li class="turbo-icon"></li>
+                <li class="turbo">turbotax</li>
+            </ul>
+        </div>
+        <div class="form">
+            <div class="form-title">
+                <p>Create an Intuit account</p>
+                <p class="more">One account for everything Intuit, including Mint. </p>
+                <a href="" class="">Learn more</a>
```
Comment (pelzolga123) [inline index.html]: It's better to delete empty class. `href` is missing `#`.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
### Important note for next project: pull request should display most of your commit's, master branch should have just few commits with initial setup.  Everything else should be pushed to feature branch and should not be merged to master until pull request is approved. 

Here is list of changes, you need to make before this pull request can be approved:
* [ ] `Phone` input should have type `tel`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

* [ ] Icons are not displaying correctly, it needs to be fixed:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: index.html (hunk 1 above)
Comment (maosan132) [reply index.html]: fixed, thanks

**Comment by maosan132** [comment]
I sent a code review request, and have a question: why aren´t my partner
receiving this email too?  meme_es@hotmail.com

Thank you

**Review by ebukaume — APPROVED** [review]
Hi @maosan132  and @meme-es 

You guys have done well with this project. **You can now go ahead and merge it.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**